source/api_cc/include/DeepPot.h

```diff
@@ -260,6 +260,14 @@ class DeepPot {
     assert(inited);
     return ntypes;
   };
+  /**
+   * @brief Get the number of types with spin.
+   * @return The number of types with spin.
+   **/
+  int numb_types_spin() const {
```

Comment:
This new public method is not tested. They should have a test so they can be migrate to the C API